resources/chefdk/msi/source.wxs.erb

```diff
@@ -12,24 +12,57 @@
           Version="$(var.VersionNumber)" Manufacturer="!(loc.ManufacturerName)" UpgradeCode="$(var.UpgradeCode)">
 
     <!--
-      Define the minimum supported installer version (2.0).
-      The install should be done for the whole machine not just the current user
+      Minimum installer version (5.0) - Window 7/Server 2008 R2 and above.
+      The install scope is per machine, not the current user
     -->
-    <Package InstallerVersion="200" InstallPrivileges="elevated"
+    <Package InstallerVersion="500" InstallPrivileges="elevated"
```

Comment:
As noted on the omnibus PR, we should avoid bumping requirements until they're hard, because some folks are still using older "unsupported" platforms.

https://github.com/chef/omnibus/pull/565/files#r44336427